community/py-magic/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-magic
+_pkgname=${pkgname#py-*}
+_pkgreal=${pkgname/py/python}
+pkgver=0.4.12
+pkgrel=0
+pkgdesc="A python wrapper for libmagic"
+url="https://pypi.python.org/pypi/python-magic/"
+arch="noarch"
+license="MIT"
+depends="python libmagic"
+makedepends="python2-dev python3-dev py-setuptools"
+install=""
+options="!strip"
```

Comment:
If it does not contain any binaries, then stripping is not performed anyway; I don’t see a point in explicit disable.